src/detours.cpp

```diff
@@ -138,14 +138,14 @@ namespace rhost {
 #ifdef _WIN32
             MH_Initialize();
 
-            MH_CreateHook(&MessageBoxW, &DetourMessageBoxW, reinterpret_cast<LPVOID*>(&pMessageBoxW));
-            MH_CreateHook(&MessageBoxA, &DetourMessageBoxA, reinterpret_cast<LPVOID*>(&pMessageBoxA));
+            MH_CreateHook((void*)&MessageBoxW, (void*)&DetourMessageBoxW, reinterpret_cast<LPVOID*>(&pMessageBoxW));
```

Comment:
static_cast won't work - data and function pointers are not convertible in standard C. I think reinterpret_cast might work, let me give it a try.